src/components/autocomplete/js/autocompleteController.js

```diff
@@ -106,7 +106,12 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
         left   = hrect.left - root.left,
         width  = hrect.width,
         offset = getVerticalOffset(),
+        position = $attrs.mdDropdownPosition,
```

Comment:
Can you use `$scope.dropdownPosition` to support values which can change during runtime?